framework/doc/content/source/times/FunctorTimes.md

```diff
@@ -0,0 +1,15 @@
+# FunctorTimes
+
+!syntax description /Times/FunctorTimes
+
+The functor and its factor will be evaluated in the element containing the (0,0,0) point, the latter of which should lay
```

Comment:
This seems bizarre, but maybe I'll see the logic of it later

Reply:
well we need an element to evaluate a functor.

maybe there should be a "default argument" that does not involve anything (no face, elem, side, node) to functors?

Reply:
An "Anywhere" spatial argument that only some derived functors can handle does sound like a logical idea to me